apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerParameter.cs

```diff
@@ -160,7 +161,25 @@ internal object GetValidatedValue()
         /// </summary>
         /// <param name="options">Options to configure the <c>SpannerDbType</c> for this <c>SpannerParameter</c>.</param>
         /// <returns>The configured <see cref="SpannerDbType"/>.</returns>
-        internal SpannerDbType GetConfiguredSpannerDbType(SpannerConversionOptions options) => SpannerDbType;
+        internal SpannerDbType GetConfiguredSpannerDbType(SpannerConversionOptions options)
+        {
+            // Only if SpannerDbType of parameter is not explicitly provided by user.
+            if (_spannerDbType == null)
+            {
+                if (Value is decimal && options != null)
```

Comment:
Will options ever be null here? How hard would it be to ensure it's never null?

Reply:
I will make the change to ensure options are not null.